src/components/AccountDrawer/SmallBalanceToggle.tsx

```diff
@@ -1,30 +1,21 @@
-import { Trans } from '@lingui/macro'
-import Row from 'components/Row'
-import Toggle from 'components/Toggle'
+import { t } from '@lingui/macro'
```

Comment:
nit: If you're trying to get it in faster, this refactor could be a separate PR.